.golangci.yml

```diff
@@ -62,13 +62,12 @@ linters-settings:
       - Prefix(github.com/karmada-io/karmada)
   revive:
     rules:
-      # Disable if-return as it is too strict and not always useful.
-      # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#if-return
-      - name: if-return
-        disabled: true
       - name: superfluous-else
         arguments:
           - preserveScope
+      - name: exported
+        arguments:
+          - disableStutteringCheck
```

Comment:
I am not sure whether we need this, if this is enabled, it will have following error:
```
pkg/karmadactl/top/top_pods.go:50:6: exported: type name will be used as top.TopPodOptions by other packages, and that stutters; consider calling this PodOptions (revive)
type TopPodOptions struct {
     ^
```

I prefer to disable it.

Reply:
Less than 10.